roles/openshift_master/templates/native-cluster/atomic-openshift-master-api.j2

```diff
@@ -4,6 +4,6 @@ CONFIG_FILE={{ openshift_master_config_file }}
 # Proxy configuration
 # Origin uses standard HTTP_PROXY environment variables. Be sure to set
 # NO_PROXY for your master
-#NO_PROXY=master.example.com
-#HTTP_PROXY=http://USER:PASSWORD@IPADDR:PORT
-#HTTPS_PROXY=https://USER:PASSWORD@IPADDR:PORT
+NO_PROXY={{ openshift.common.noproxy }}
+HTTP_PROXY={{ openshift.common.http_proxy }}
+HTTPS_PROXY={{ openshift.common.https_proxy }}
```

Comment:
If we create a template for atomic-openshift-master, then we should just have a single template and somehow create the right OPTIONS line depending on which file we are writing out...